rib-bible.md

```diff
@@ -525,14 +525,217 @@ GitHub:
 ```toml
 repos = [
 "rust-bitcoin/rust-bitcoin",
+"rust-bitcoin/rust-secp256k1",
 "rust-bitcoin/rust-miniscript",
+"rust-bitcoin/rust-wallet",
 "rust-bitcoin/rust-bitcoincore-rpc",
+"rust-bitcoin/rust-bech32-bitcoin",
+"rust-bitcoin/murmel",
+"rust-bitcoin/rust-bip39",
+"rust-bitcoin/bitcoin_hashes",
+"jean-airoldie/zeromq-src-rs",
+]
+```
+
+## BDK
+
+Media:
+
+- https://bitcoindevkit.org
+
+GitHub:
+
+- https://github.com/bitcoindevkit
+
+```toml
+repos = [
+"bitcoindevkit/bdk",
+"bitcoindevkit/bdk-cli",
+"bitcoindevkit/bdk-ffi",
+"bitcoindevkit/bdk-hwi",
+"rust-electrum-client",
+]
+```
+
+## LDK
+
+Media:
+
+- https://lightningdevkit.org
+
+GitHub:
+
+- https://github.com/lightningdevkit
+
+```toml
+repos = [
 "lightningdevkit/rust-lightning",
 "lightningdevkit/ldk-sample",
 "lightningdevkit/ldk-c-bindings",
 ]
 ```
 
+## Sapio
+
+Media:
+
+- https://learn.sapio-lang.org/
+
+GitHub:
+
+- https://github.com/sapio-lang
+
+```toml
+repos = [
+"sapio-lang/sapio",
+]
+```
+
+## LNP
+
+Media:
+
+- https://www.lnp-bp.org
+
+GitHub:
+
+- https://github.com/LNP-BP
+
+```toml
+repos = [
+"LNP-BP/lnp-core",
+"LNP-BP/lnp-node",
+"LNP-BP/bp-node",
+"LNP-BP/bp-core",
+"LNP-BP/client_side_validation",
+"LNP-BP/descriptor-wallet",
+"LNP-BP/rust-amplify",
+"LNP-BP/ln-types",
+"LNP-BP/invoices",
+"LNP-BP/rust-psbt",
+"LNP-BP/lnp-sdk",
+"LNP-BP/lbx",
+]
+```
+
+## RGB
+
+Media:
+
+- https://www.rgbfaq.com
+- https://rgbex.io
+
+GitHub:
+
+- https://github.com/rgb-org
```

Comment:
This link is 404. Can you fix it here and in the markdown template?

Reply:
Updated